lib/commands/new/transpilers/typescript.js

```diff
@@ -24,11 +20,6 @@ module.exports = function(project) {
     'gulp-typescript',
     'gulp-tslint',
     'tslint',
-    'typescript',
```

Comment:
shouldn't `@types/jasmine` and `@types/node` be added to the list of dev dependencies here? for them to be included in the `project.json` ready for when `installDependencies` is called in `project-template.js`?